sdk/src/main/java/com/eegeo/mapapi/widgets/RouteViewOptions.java

```diff
@@ -37,6 +38,17 @@ public RouteViewOptions color(int color) {
         return this;
     }
 
+    /**
+     * Sets the current step color of the RouteView's polylines as a 32-bit ARGB color. The default value is opaque red (0xff000000).
+     *
+     * @param color The color to use.
+     * @return The RouteViewOptions object on which the method was called, with the new color set.
+     */
+    public RouteViewOptions activeColor(int color) {
```

Comment:
Could be named better. Active color does not show which part is active. Maybe traversed path and remaining path or forward path would be more suitable. Open for suggestions here. Please also make sure that the naming convention matches on iOS.

Reply:
relevant changes have been made.